database/migrations/2017_05_15_184344_add_soft_delete_to_tagging_tagged_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddSoftDeleteToTaggingTaggedTable extends Migration
```

Comment:
Why do we need this? the `tag()` and `unTag()` should handle the addition and deletion of tags.